dlt/common/destination/capabilities.py

```diff
@@ -30,6 +30,8 @@
 # insert_values - insert SQL statements
 # sql - any sql statement
 TLoaderFileFormat = Literal["jsonl", "typed-jsonl", "insert_values", "parquet", "csv"]
+TLoaderParallelismStrategy = Literal["parallel", "table_sequential", "sequential"]
```

Comment:
hmm I think we are using dashes not underscores in other places? surely the above both are present :)